bigtable/tests/Makefile

```diff
@@ -0,0 +1,26 @@
+# A simple makefile to test the `install` target.
```

Comment:
Because neither #45 nor #44 are done yet.  Writing a simple Makefile that is hard-coded is easier than writing a CMakeLists.txt that is kind of hard-coded.  Happy to change this if we agree on the rest of the proposal though.

Reply:
Added a comment to declare what we should do here.